pkg/queue/readiness/probe_test.go

```diff
@@ -102,7 +102,8 @@ func TestExecHandler(t *testing.T) {
 		Handler: corev1.Handler{
 			Exec: &corev1.ExecAction{
 				Command: []string{"echo", "hello"},
-			}},
+			},
+		},
```

Comment:
Wanna drop this diff? Seems completely unrelated.

Reply:
huh, how on earth did that slip in?! yep, dropped